src/main/java/com/epam/lowcost/DAO/UserDAOImpl.java

```diff
@@ -55,6 +59,21 @@ public User getById(long userId) {
         return user;
     }
 
+
+    @Override
+    public User findByEmail(String log, String pass) {
+
+
+        try {
+            User user = jdbcTemplate.queryForObject("SELECT * FROM USERS WHERE email=? and password=?", new UserRowMapper(), log, pass);
```

Comment:
why you create rowMapper everytime ?

Reply:
Changed to have one UserRowMapper instance for application.